automatic/googlechrome/update.ps1

```diff
@@ -1,8 +1,7 @@
 ﻿import-module au
 import-module "$PSScriptRoot\..\..\scripts\au_extensions.psm1"
 
-$releases = 'http://omahaproxy.appspot.com/all?os=win&amp;channel=stable'
-$paddedUnderVersion = '57.0.2988'
```

Comment:
This `$paddedUnderVersion` variable should not be removed.

Reply:
Whoops! Didn't realize I removed that. Total accident. One sec, I'll fix it.

Reply:
Fixed it.